### PR TITLE
[Tests] reduce the model size in the ddpm fast test

### PR DESCRIPTION
**Pull request by ariG23498**
# What does this PR do?

Reduces the model sizes in the DDPM fast tests.

Tagging Issue: https://github.com/huggingface/diffusers/issues/7677

CC: @sayakpaul for visibility.

### CODE REVIEW

File: tests/pipelines/ddpm/test_ddpm.py [hunk 1]
```diff
@@ -30,8 +30,9 @@ class DDPMPipelineFastTests(unittest.TestCase):
     def dummy_uncond_unet(self):
         torch.manual_seed(0)
         model = UNet2DModel(
-            block_out_channels=(32, 64),
-            layers_per_block=2,
+            block_out_channels=(8, 8),
```
Comment (sayakpaul) [inline tests/pipelines/ddpm/test_ddpm.py:33]: Can we do something like (4, 8)?

File: tests/pipelines/ddpm/test_ddpm.py [hunk 2]
```diff
@@ -30,8 +30,9 @@ class DDPMPipelineFastTests(unittest.TestCase):
...
+            layers_per_block=1,
+            norm_num_groups=8,
             sample_size=32,
```
Comment (sayakpaul) [inline tests/pipelines/ddpm/test_ddpm.py:36]: Can we also reduce this element?

**Review by sayakpaul — COMMENTED** [review]
Thanks for the PR. Left some comments.

File: tests/pipelines/ddpm/test_ddpm.py (hunk 1 above)
Comment (ariG23498) [reply tests/pipelines/ddpm/test_ddpm.py]: Addressed!

File: tests/pipelines/ddpm/test_ddpm.py (hunk 2 above)
Comment (ariG23498) [reply tests/pipelines/ddpm/test_ddpm.py]: Addressed!

**Review by sayakpaul — APPROVED** [review]
LGTM! Thanks.

**Pull request merged**